csharp/Platform.Collections/Segments/Walkers/AllSegmentsWalkerBase[T].cs

```diff
@@ -31,7 +31,7 @@ public abstract class AllSegmentsWalkerBase<T> : AllSegmentsWalkerBase<T, Segmen
         /// <returns>
         /// <para>An instance of the <see cref="Segment{T}"/> class.</para>
         /// <para>Экземпляр класса <see cref="Segment{T}"/> .</para>
```

Comment:
An extra space before the dot, as well as on line 17

Reply:
Also an extra space on line 21

Reply:
Remove extra space